functions.js

```diff
@@ -10,6 +10,24 @@ const getAllStations = async (req, res) => {
     }
 }
 
+const calculateCosts = (originZone, destinationZone) => {
+
+    let cost = 0
```

Comment:
Couldn't you just set the cost to 3.99 in the first place, since you're adding that in all cases? Then you can remove the first branch in the conditional.